### PR TITLE
fix : fix CI pipelines with updated enviroment

### PR DESCRIPTION
**Pull request by Ming-Yan**
- update `readme.md`

- update enviroment file

- remove redudant env file

fix: update workflows

- remove coffea.hist

- correct DY workflows

feat: rename SFs name, issues for coffea

feat : add global flags to runner.py

fix: fix error with no correction case

feat : add corrections to workflow

- add correctiosn to workflow

- minor fix on parameters

### CODE REVIEW

File: src/BTVNanoCommissioning/utils/correction.py [hunk 1]
```diff
@@ -218,10 +218,14 @@ def eleSFs(ele, campaign, path):
     with contextlib.ExitStack() as stack:
         real_paths = [
             stack.enter_context(importlib.resources.path(_ele_path, f))
-            for f in path.values()
+            for f in path.values()  # if "ele" in path.keys()
```
Comment (demuller) [inline src/BTVNanoCommissioning/utils/correction.py:221]: comment could be removed

File: src/BTVNanoCommissioning/utils/correction.py [hunk 2]
```diff
@@ -238,15 +242,19 @@ def eleSFs(ele, campaign, path):
...
 def muSFs(mu, campaign, path):
-    _ele_path = f"BTVNanoCommissioning.data.LSF.{campaign}"
+    _mu_path = f"BTVNanoCommissioning.data.LSF.{campaign}"
     ext = extractor()
     with contextlib.ExitStack() as stack:
         real_paths = [
-            stack.enter_context(importlib.resources.path(_ele_path, f))
-            for f in path.values()
+            stack.enter_context(importlib.resources.path(_mu_path, f))
+            for f in path.values()  # if "mu" in path.keys()
```
Comment (demuller) [inline src/BTVNanoCommissioning/utils/correction.py:250]: comment could be removed

**Comment by demuller** [comment]
Hi @Ming-Yan  the PR looks overall good to me, waiting for review from @andrzejnovak

**Comment by andrzejnovak** [comment]
Lgtm, +1

**Pull request merged**